src/i18n/Localizable.xcstrings

```diff
@@ -23030,6 +23149,7 @@
       }
     },
     "Stick" : {
+      "comment" : "a bit ambiguous word. example needed :(",
```

Comment:
"Stick" is like "Pad" but instead of directional keys, here it is a joystick

Reply:
transliterated "stick". is this a virtual or physical joystick that can move screen around, like a Pad?  
I found out that the term "virtual stick" is commonly used in Korean, so just transliterated word "stick" will do, I think.